src/js/ripple/amount.js

```diff
@@ -975,29 +937,25 @@ Amount.prototype.not_equals_why = function(d, ignore_issuer) {
     return 'Not an Amount';
   }
 
-  var result = false;
-
   if (!this.is_valid() || !d.is_valid()) {
-    result = 'Invalid amount.';
+    return 'Invalid amount.';
```

Comment:
I switched to returns because of the long-term goal of minimizing the use of variables (one of the todo items on the pull request for the javascript style guide). I'll refactor to remove else's.